getting-started/README.md

```diff
@@ -8,6 +8,10 @@ Optimism is [EVM equivalent](https://medium.com/ethereum-optimism/introducing-ev
 Therefore, we the differences between Optimism development and Ethereum development are minor.
 But a few differences [do exist](https://community.optimism.io/docs/developers/build/differences/#).
 
+#### See video
+
+[![See a video with this information](https://img.youtube.com/vi/_Y6CwsYgqwI/default.jpg)](https://youtu.be/_Y6CwsYgqwI)
```

Comment:
```suggestion
If you prefer watching tutorials rather than reading them, we have [![this great video covering the getting started content](https://img.youtube.com/vi/_Y6CwsYgqwI/default.jpg)](https://youtu.be/_Y6CwsYgqwI)
```

Reply:
IDK if that is formatted correctly, but something along this line.

Reply:
Not formatted correctly, let me explain.

`![<text>](<url>}` means show the picture in the URL, and only if the user is using assistive technology (for example, because they are blind) use the text. In this case, the image is itself the "text" in a link to the video.

I fixed it, ready for merge IMAO.